docs/source/cli/slide-out.rst

```diff
@@ -6,9 +6,11 @@ slide-out
 
 .. code-block:: shell
 
-    git machete slide-out [-d|--down-fork-point=<down-fork-point-commit>] [--delete] [-M|--merge] [-n|--no-edit-merge|--no-interactive-rebase] [<branch> [<branch> [<branch> ...]]]
+    git machete slide-out [-d|--down-fork-point=<down-fork-point-commit>] [--delete] [--all-merged] [-M|--merge] [-n|--no-edit-merge|--no-interactive-rebase] [<branch> [<branch> [<branch> ...]]]
```

Comment:
Makes sense to use `--all-...` prefix indeed to avoid confusion with `--merge`, good spot